nexus-sdk-js/src/utils/events.ts

```diff
@@ -0,0 +1,35 @@
+// @ts-ignore
+import EventSource = require('eventsource');
+import store from '../store';
+
+// TODO: speak to backend to figure out token
+const getHeaders = (): EventSourceInit => {
```

Comment:
I thought we already had a generic method to add auth token to a request. If not, better make it something generic inside utils/http.ts

Reply:
Headers for an` HTTP` request or `EventSource` are completely different unfortunately. There aren't really header for ES, you just give it a config: https://developer.mozilla.org/en-US/docs/Web/API/EventSource/EventSource

That's why I put a wee TODO there, making sure we find a solution with the backend team

Reply:
I see